src/containers/Tenant/Diagnostics/Overview/TableInfo/TableInfo.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {useMemo} from 'react';
+import cn from 'bem-cn-lite';
+
+import type {TEvDescribeSchemeResult} from '../../../../../types/api/schema';
+
+import {InfoViewer} from '../../../../../components/InfoViewer';
+
+import {getEntityName} from '../../../utils';
+
+import {prepareTableInfo} from './prepareTableInfo';
+
+import './TableInfo.scss';
+
+const b = cn('ydb-diagnostics-table-info');
+
+interface TableInfoProps {
+    data?: TEvDescribeSchemeResult;
+}
+
+export const TableInfo = ({data}: TableInfoProps) => {
+    const entityName = getEntityName(data?.PathDescription);
+
+    const {
+        generalTableInfo = [],
+        tableStatsInfo = [],
+        tabletMetricsInfo = [],
+        partitionConfigInfo = [],
+    } = useMemo(() => prepareTableInfo(data), [data]);
+
+    if (
```

Comment:
let's put it into a helper function `checkIsEmptyTableInfo`.
Why `tableStatsInfo` is flatten, but other arrays are not?

Reply:
`tableStatsInfo` is flatten, because it is array of arrays (`[][]`).